frontend/app/src/pages/superadmin/statistics/Statistics.spec.tsx

```diff
@@ -16,34 +16,22 @@ beforeAll(() => {
 /**
  * @jest-environment jsdom
  */
+
+const mockMetrics = {
+  bounties_posted: 100,
+  bounties_paid: 50,
+  sats_posted: 5000,
+  sats_paid: 2500,
+  bounties_paid_average: 75,
+  sats_paid_percentage: 50,
+  average_paid: 10,
+  average_completed: 1
+};
+
 describe('Statistics Component', () => {
   nock(user.url).get('/person/id/1').reply(200, {});
-  test('display about view with extras', () => {
-    const hardcodedWords = [
-      'Bounties',
-      '200',
-      'Total Bounties Posted',
-      '78',
-      'Bounties Assigned',
-      '136',
-      '100%',
-      '22536',
-      'Total Sats Posted',
-      '13625',
-      'Sats Paid',
-      '3 Days',
-      'Avg Time to Paid',
-      '48%',
-      'Paid'
-    ];
 
-    render(<Statistics />);
-    const bountiesPaid = screen.getAllByText('Bounties Paid');
-    if (bountiesPaid) {
-      expect(bountiesPaid[0]).toBeInTheDocument();
-    }
-    for (let i = 0; i < hardcodedWords.length; i++) {
-      expect(screen.queryByText(hardcodedWords[i])).toBeInTheDocument();
-    }
+  test('display about view with extras', () => {
+    render(<Statistics metrics={mockMetrics} />);
```

Comment:
I dont see any expect statements we should be still testing for things

lets test for the mockMetrics to show up and specific words like `Bounties, Total Bounties Posted and ect` to be showing up aswell

Reply:
Sure will add those and update this